<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatePracticeDiaLog</class>
 <widget class="QDialog" name="CreatePracticeDiaLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fff;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Source Sans 3 Medium</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #000;</string>
     </property>
     <property name="text">
      <string>Create Practice</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>275</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgba(0,0,0,0.2);
border-radius: 4px;
padding-left: 10px;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>75</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #000;</string>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="categoryComboBox">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>275</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>0</y>
        <width>75</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
      <property name="text">
       <string>Category:</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <widget class="QRadioButton" name="levelRadioButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {width: 20px; height: 20px;}</string>
      </property>
      <property name="text">
       <string>Config level</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="randomLevel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {width: 20px; height: 20px;}</string>
      </property>
      <property name="text">
       <string>Random</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="levelGroup" native="true">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>75</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
      <property name="text">
       <string>Level 1:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>0</y>
        <width>75</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
      <property name="text">
       <string>Level 3:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="level3SpinBox">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>0</y>
        <width>75</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
      <property name="text">
       <string>Level 5:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="level5SpinBox">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>30</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
     </widget>
     <widget class="QSpinBox" name="level2SpinBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>75</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
      <property name="text">
       <string>Level 2:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="level4SpinBox">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>30</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>0</y>
        <width>75</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
      <property name="text">
       <string>Level 4:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="level1SpinBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <widget class="QSpinBox" name="timeSpinBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3</family>
       </font>
      </property>
      <property name="text">
       <string>Time Limit:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submitPushButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>131</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans 3 SemiBold</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">#submitPushButton {
	background-color:  rgb(0, 191, 154);
	border: none;
	color: #fff;
	border-radius: 4px;
}

#submitPushButton {
	background-color:  rgba(0, 191, 154, 0.9);
}</string>
      </property>
      <property name="text">
       <string>Let's Practice</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
